test/TaxesWidget.js

```diff
@@ -1,5 +1,13 @@
 root				: '#taxes',
 toggleLink			: '#taxes summary',
-taxesList			: '#taxes table',
+taxesList			: '#taxes .category',
 randomTax			: '#taxes [data-source]',
+impreciseTaxValue  : '#taxes .imprecise [data-source]',
+impreciseTaxTooltipText  : '#taxes .imprecise .explanation div',
 createTestButton	: '#createTest',
+
+hoverOverImpreciseTax: function() {
+    return this.impreciseTaxValue.then(function(resultElement) {
```

Comment:
Call the passed element `impreciseTaxValue` rather than `resultElement`, so its name is consistent.